src/app/home/home.scss

```diff
@@ -1,12 +1,13 @@
 .home-container {
-  margin: 0 8rem;
+  margin: 0 7rem;
   margin-top: 2rem;
 }
 
-.mySlides {display: none}
-img {vertical-align: middle;}
+.bannerSlides {display: none}
 
-/* Slideshow container */
+img { vertical-align: middle; }
```

Comment:
Where did you handle Image Scaling? https://www.geeksforgeeks.org/resize-image-proportionally-with-css/